docs/examples.md

```diff
@@ -129,9 +129,21 @@ Example at [examples/kotlin](https://github.com/bazelbuild/rules_nodejs/tree/sta
 
 ### Protocol Buffers and gRPC
 
-Note: this is considered a "labs" feature in rules_nodejs, so support and stability are not great. gRPC is still a WIP.
+Note: this is under active development. Come chat in the #javascript channel on Slack to get the latest.
+Support and stability are not great but expected to improve.
 
-See [examples/protocol_buffers](https://github.com/bazelbuild/rules_nodejs/tree/stable/examples/protocol_buffers)
+There are many alternative implementations for protobuf and RPC.
```

Comment:
nit: GRPC rather than RPC

Reply:
Actually this is intentional, there are other RPC transports that are proto compatible. From the proto team's perspective, gRPC is just one option (and for example protobufjs produces service stubs that are transport-agnostic)